src/pytest_html/scripts/main.js

```diff
@@ -62,10 +62,15 @@ const renderContent = (tests) => {
 
 const renderDerived = (tests, collectedItems, isFinished) => {
     const currentFilter = getVisible()
-    possibleResults.forEach(({ result, label }) => {
+    possibleResults.forEach(({ result, label }, index) => {
         const count = tests.filter((test) => test.result.toLowerCase() === result).length
```

Comment:
Why include index?

Reply:
Good catch, it's a left-over from a previous solution. Will remove.